docs/building-acc-target-deps.md

```diff
@@ -22,6 +22,24 @@ You will need to build two versions of the libraries:
 
 The Host and Target libraries must be the same version.
 
+## Prerequisites
+
+It is assumed at the target-dependent's SDE (DPDK SDE or ES2K IPU SDE) dependencies have been installed on the system. The SDE's dependencies can be found in `sde/tools/setup/install_dep.py` file. Some of those dependencies also apply to this module. 
+
+Install the following dependencies for building stratum-deps.
+
+For Fedora:
+
+```bash
+dnf install -y openssl openssl-devel
```

Comment:
Should we add a note specifying recommended openssl version(3.1)?

Reply:
Should we have sudo in these commands?

Reply:
hmmm I have followed similar commands as documented elsewhere. If user is non-admin, will need to have sudo privileges (should be self-evident to user). In addition a lot of user's functionality on platform assumes root privileges, so going to leave the `sudo` out of the install commands.

Re the OpenSSL 3.x -- theres a note below and also a link to the OpenSSL upgrade guide document -- I think thats sufficient.